### PR TITLE
[Release] Linux_SQL

### PR DESCRIPTION
**Pull request by HongtingYin**
https://www.notion.so/417963db3cdf4ac0b4c36a9b72950b2c?v=88fd7654e97c46cc9c81cbd37307123e&p=3ea0302938e04b8c83edb4aa8d58640e

https://www.notion.so/417963db3cdf4ac0b4c36a9b72950b2c?v=88fd7654e97c46cc9c81cbd37307123e&p=bf3c4d4c187e473eb8ca43c27d63edb8

https://www.notion.so/417963db3cdf4ac0b4c36a9b72950b2c?v=88fd7654e97c46cc9c81cbd37307123e&p=c59d1985087140ea99d6bac3a2e3e24d

https://www.notion.so/417963db3cdf4ac0b4c36a9b72950b2c?v=88fd7654e97c46cc9c81cbd37307123e&p=faa05bfd04eb457b9bd93678e58d0467

https://www.notion.so/417963db3cdf4ac0b4c36a9b72950b2c?v=88fd7654e97c46cc9c81cbd37307123e&p=a05ef5b644f940bf84de94889b96bead

### CODE REVIEW

File: linux_sql/README.md [hunk 1]
```diff
@@ -1,2 +1,90 @@
...
+
+## Introduction
+This cluster monitoring agent is used to record the hardware specifications of each node and
+ monitor node resource usage in real-time. The collected data can be used to generate report
+s by the infrastructure team which helps them to make better resource planning in the future
+.
+
+## Architecture & Design
+
+![diagram](./assets/architecture.png)
```
Comment (jarviscanada) [inline linux_sql/README.md:11]: image

File: linux_sql/README.md [hunk 2]
```diff
@@ -1,2 +1,90 @@
...
+
```
Comment (jarviscanada) [inline linux_sql/README.md:12]: use a title here to make it more readable 

e.g.
`### Tables`

File: linux_sql/README.md [hunk 3]
```diff
@@ -1,2 +1,90 @@
...
+```sh
+# Edit crontab jobs
+crontab -e
+# Add following command and host_usage.sh will be running periodically
+* * * * * bash /home/centos/dev/jarvis_data_eng_Hongting/linux_sql/scripts/host_usage.sh localhost 5432 host_agent postgres docker > /tmp/host_usage.log
+# List crontab jobs
+crontab -l
+```
+
+## Improvements
```
Comment (jarviscanada) [inline linux_sql/README.md:86]: in general, you need to read more content for this README file. I attached a sample README file in the Notion ticket.

**Review by jarviscanada — APPROVED** [review]
fix all comments and merge

File: linux_sql/README.md (hunk 1 above)
Comment (HongtingYin) [reply linux_sql/README.md:11]: Thanks. Updated!

File: linux_sql/README.md (hunk 2 above)
Comment (HongtingYin) [reply linux_sql/README.md:12]: Thanks. Updated!

File: linux_sql/README.md (hunk 3 above)
Comment (HongtingYin) [reply linux_sql/README.md:86]: Thanks. Updated!

**Pull request merged**